mongo/options/createcollectionoptions.go

```diff
@@ -26,6 +26,44 @@ func (d *DefaultIndexOptions) SetStorageEngine(storageEngine interface{}) *Defau
 	return d
 }
 
+// TimeSeriesOptions specifies options on a time-series collection.
```

Comment:
There are no real Docs to link to for time-series collections yet, but I wonder if I should add more documentation here on how they work. Let me know if the existing comments don't paint a clear picture.

Reply:
I assume that the server docs will cover this, so I think it'll be fine

Reply:
I think the reference for the options will fall under the `create` command documentation. I don't see any existing links to the page. A blurb on `Database.CreateCollection` may be helpful:

```
// For more information about the command, see https://docs.mongodb.com/manual/reference/command/create/.
```

The 5.0 docs are not ready for time series. FWIW you can see them by changing the URL to `v5.0`: https://docs.mongodb.com/v5.0/reference/command/create/

Reply:
Added that line as documentation to `database#CreateCollection`.